x-pack/platform/plugins/shared/index_management/test/scout/ui/tests/home_page.spec.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout';
+import { test } from '../fixtures';
+
+test.describe('Home page', { tag: ['@ess'] }, () => {
+  test.beforeEach(async ({ pageObjects, browserAuth }) => {
+    await browserAuth.loginAsIndexManagementUser();
+    await pageObjects.indexManagement.goto();
+  });
+
+  test('Loads the app and renders the indices tab by default', async ({
+    pageObjects,
+    log,
+    page,
+  }) => {
+    await log.debug('Checking for section heading to say Index Management.');
+
+    const headingText = await pageObjects.indexManagement.sectionHeadingText();
+    expect(headingText).toBe('Index Management');
+
+    // Verify url
+    expect(page.url()).toContain('/indices');
+
+    // Verify content
+    await expect(page.testSubj.locator('indicesList')).toBeVisible();
+    await expect(page.testSubj.locator('reloadIndicesButton')).toBeVisible();
+  });
+
+  test('Indices - renders the indices tab', async ({ pageObjects, page }) => {
+    // Navigate to the indices tab
+    await pageObjects.indexManagement.changeTabs('indicesTab');
+
+    // Verify url
+    expect(page.url()).toContain('/indices');
+
+    // Verify content - wait for table to be visible
+    await expect(page.testSubj.locator('indexTable')).toBeVisible();
+  });
+
+  test('Indices - can create an index', async ({ pageObjects, esClient, log }) => {
+    const testIndexName = `index-test-${Math.random()}`;
+
+    await pageObjects.indexManagement.clickCreateIndexButton();
+    await pageObjects.indexManagement.setCreateIndexName(testIndexName);
+    await pageObjects.indexManagement.setCreateIndexMode('Lookup');
+    await pageObjects.indexManagement.clickCreateIndexSaveButton();
+    await pageObjects.indexManagement.expectIndexToExist(testIndexName);
+
+    // Cleanup
+    try {
+      await esClient.indices.delete({ index: testIndexName });
+    } catch (e: any) {
+      log.debug(`Index cleanup failed for ${testIndexName}: ${e.message}`);
+    }
```

Comment:
_question_: having it in the test might not work if test fails, right? Do you want to move to `afterEach` hook?